packages/forms-web-app/src/controllers/rule-6/enter-code.js

```diff
@@ -51,6 +51,12 @@ const getEnterCodeR6 = (views) => {
 
 		const email = getSessionEmail(req.session, false);
 
+		const isRule6User = await isRule6UserByEmail(req, email);
+
+		if (!isRule6User) {
+			return renderEnterCodePage(`/${views.EMAIL_ADDRESS}`);
```

Comment:
I think this gives away that a user has rule6 status
See last scenario on A2-1202
User should be able to post a code but it should just always fail, not redirect to email
But we don't want them to receive an email with a code, so just render page without generating